index.html

```diff
@@ -117,6 +117,227 @@ <h3 class="hero-section__subtitle">I'm a software developer</h3>
+        <div class="projects-card">
+          <img
+            src="src/assets/media/img-placeholder.png"
+            alt="template will be replaced by a project soon"
+            class="projects-card__img"
+          />
+          <div class="projects-card__title--wrapper">
+            <p class="projects-card__title">Multi-Post Stories</p>
+            <p class="projects-card__title">Gain+Glory</p>
+          </div>
+          <div class="projects-card__tech-stack--wrapper">
+            <p class="projects-card__tech-stack">Ruby on Rails</p>
+            <p class="projects-card__tech-stack">CSS</p>
+            <p class="projects-card__tech-stack">JavaScript</p>
+            <p class="projects-card__tech-stack">HMTL</p>
+          </div>
+          <a
+            href="#"
+            class="projects-card__btn"
+            target="_blank"
+            rel="noo
+           noreferrer"
+            >See project</a
+          >
+        </div>
+      </section>
+    </main>
+    <section class="about-section">
+      <div class="about-section__about-me--wrapper">
+        <h2 class="about-me__title">About me</h2>
+        <p class="about-me__intro">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+        <a
+          href="#"
+          class="about-me__btn"
+          target="_blank"
+          rel="noopener noreferrer"
+          >Get my resume</a
+        >
+      </div>
+      <div class="about-section__tech-stack-container container">
+        <div class="about-section__tech-stack">
+          <img
+            src="src/assets/media/sections-illustrations/tech-stack-type-illustrations/tech-stack-grid-illustration-languages.png"
+            alt="languages icon"
+            class="tech-stack__description-logo"
+          />
+          <p class="tech-stack__type">Languages</p>
+          <div class="tech-stack__title--wrapper">
+            <p class="tech-stack__title">JavaScript</p>
+            <p class="tech-stack__title">Ruby</p>
+            <p class="tech-stack__title">HTML</p>
+            <p class="tech-stack__title">CSS</p>
+          </div>
+        </div>
+        <div class="about-section__tech-stack">
+          <img
+            src="src/assets/media/sections-illustrations/tech-stack-type-illustrations/tech-stack-grid-illustration-frameworks.png"
+            alt="frameworks icon"
+            class="tech-stack__description-logo"
+          />
+          <p class="tech-stack__type">Frameworks</p>
+          <div class="tech-stack__title--wrapper">
+            <p class="tech-stack__title">React.js</p>
+            <p class="tech-stack__title">Ruby on Rails</p>
+            <p class="tech-stack__title">RSpec</p>
+            <p class="tech-stack__title">Capybara</p>
+            <p class="tech-stack__title">Selenium</p>
+          </div>
```

Comment:
- Kindly, use `ul` to group a list of languages, `li` per icon, and apply this for other about section language lists for semantic reasons 👌